smithy-typescript-ssdk-libs/server-common/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@aws-smithy/server-common",
+  "version": "1.0.0",
+  "description": "Base components for Smithy services",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/es/index.js",
+  "types": "./types/index.d.ts",
+  "scripts": {
+    "prepublishOnly": "yarn build:cjs && yarn build:es",
+    "pretest": "yarn build:cjs",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build:es": "tsc -p tsconfig.es.json",
+    "build": "yarn build:es && yarn build:cjs",
```

Comment:
Another thing worth discussing is the necessity of shipping ES module dist or `tsconfig.es.json` and `build:es` script. Currently, this setup will ship 2 99% similar distributions but support different modules in JS(CJS and ESM). Since this is a node-only server-side SDK,  there's no need to generate browser-friendly but older syntax ESM modules.